### PR TITLE
Robustify test_upload_spectroscopy

### PR DESCRIPTION
**Pull request by kmshin1397**
Use new baselayer driver function options with test_upload_spectroscopy.

Noticed this was failing on recent CI runs - looks like I forgot to update this test yesterday

### CODE REVIEW

**Comment by dannygoldstein** [comment]
Thanks for catching this

**Review by dannygoldstein — APPROVED** [review]

File: skyportal/tests/frontend/test_upload_spectroscopy.py [hunk 1]
```diff
@@ -23,17 +23,17 @@ def test_upload_spectroscopy(
     driver.scroll_to_element_and_click(mjd_element)
     mjd_element.send_keys('51232.')
 
-    instrument_id_element = driver.wait_for_xpath(f'//*[@id="root_instrument_id"]')
-    driver.scroll_to_element_and_click(instrument_id_element)
+    instrument_id_element = f'//*[@id="root_instrument_id"]'
+    driver.click_xpath(instrument_id_element)
```
Comment (acrellin) [inline skyportal/tests/frontend/test_upload_spectroscopy.py:27]:
```suggestion
    instrument_id_element_xpath = f'//*[@id="root_instrument_id"]'
    driver.click_xpath(instrument_id_element_xpath)
```

File: skyportal/tests/frontend/test_upload_spectroscopy.py [hunk 2]
```diff
@@ -23,17 +23,17 @@ def test_upload_spectroscopy(
...
 
-    sedm_element = driver.wait_for_xpath(f'//li[@data-value="{inst_id}"]')
-    driver.scroll_to_element_and_click(sedm_element)
+    sedm_element = f'//li[@data-value="{inst_id}"]'
+    driver.click_xpath(sedm_element, scroll_parent=True)
```
Comment (acrellin) [inline skyportal/tests/frontend/test_upload_spectroscopy.py:30]:
```suggestion
    sedm_element_xpath = f'//li[@data-value="{inst_id}"]'
    driver.click_xpath(sedm_element_xpath, scroll_parent=True)
```

File: skyportal/tests/frontend/test_upload_spectroscopy.py [hunk 3]
```diff
@@ -23,17 +23,17 @@ def test_upload_spectroscopy(
...
 
-    preview_button = driver.wait_for_xpath(f'//button[contains(.,"Preview")]')
-    driver.scroll_to_element_and_click(preview_button)
+    preview_button = f'//button[contains(.,"Preview")]'
+    driver.click_xpath(preview_button)
```
Comment (acrellin) [inline skyportal/tests/frontend/test_upload_spectroscopy.py:33]:
```suggestion
    preview_button_xpath = f'//button[contains(.,"Preview")]'
    driver.click_xpath(preview_button_xpath)
```

File: skyportal/tests/frontend/test_upload_spectroscopy.py [hunk 4]
```diff
@@ -23,17 +23,17 @@ def test_upload_spectroscopy(
...
 
-    submit_button = driver.wait_for_xpath(f'//button[contains(.,"Upload Spectrum")]')
-    driver.scroll_to_element_and_click(submit_button)
+    submit_button = f'//button[contains(.,"Upload Spectrum")]'
+    driver.click_xpath(submit_button)
```
Comment (acrellin) [inline skyportal/tests/frontend/test_upload_spectroscopy.py:36]:
```suggestion
    submit_button_xpath = f'//button[contains(.,"Upload Spectrum")]'
    driver.click_xpath(submit_button_xpath)
```

**Review by acrellin — APPROVED** [review]
Just a naming suggestion, as otherwise it sounds like the variable is an element rather than xpath string

**Pull request merged**